.github/workflows/publish_to_pypi.yml

```diff
@@ -0,0 +1,106 @@
+name: Publish xrpl-py 🐍 distribution 📦 to PyPI
+on:
+  push:
+    tags:
+        - '*'
```

Comment:
Since there is a possibility of releasing specific feature-builds (like AMM, MPT, etc), I will not incorporate this suggestion. I prefer to leave some flexibility in definition of the tags.